### DISCUSSION TITLE
debugging

### DISCUSSION THREAD

**Discussion by cooper2751** [lbbrhzn/ocpp#741] [Q&A] [answered]
Hello can any one help? don't know if I've set something incorrectly or not on install. I've been using this integration via HACS  for about 4 months and it has worked with out any problems. but daily or on HA restart i get spammed with notification.
after re reading the Docs to day and checking the logs i notices that is also very busy from ocpp as well.
below is what i get from the notification tab in HA.

``` 
Warning: charger reported NotSupported while setting WebSocketPingInterval=60
``` 

``` 
Warning: charger reported NotSupported while setting ClockAlignedDataInterval=900
```

```
Warning: charger reported NotSupported while setting MeterValuesSampledData=Energy.Active.Import.Register,Energy.Reactive.Import.Register,Energy.Active.Import.Interval,Energy.Reactive.Import.Interval,Power.Active.Import,Power.Reactive.Import,Power.Offered,Power.Factor,Current.Import,Current.Offered,Voltage,Frequency,RPM,SoC,Temperature,Current.Export,Energy.Active.Export.Register,Energy.Reactive.Export.Register,Energy.Active.Export.Interval,Energy.Reactive.Export.Interval,Power.Active.Export,Power.Reactive.Export
```

```
No feature profiles detected, defaulting to Core
```

This one reoccurs in the logs a lot

```
2023-01-14 20:48:12.075 INFO (MainThread) [ocpp] S/N********: receive message [2,"7160","Heartbeat",{}]
2023-01-14 20:48:12.077 INFO (MainThread) [ocpp] S/N********: send [3,"7160",{"currentTime":"2023-01-14T20:48:12Z"}]
```

thanks for any help.
Ben

**Comment by drc38** [comment, marked as answer]
You can dismiss and ignore the warnings.  Your charger is not fully ocpp compliant (hence the not supported messages)  but the integration will still work ok.

**Reply by cooper2751** [reply]
thank you for clarifying  i will ignore warnings now
